README.md

```diff
@@ -0,0 +1,14 @@
+# slackthemes
```

Comment:
Please remove the README file addition. I don't plan on adding one as part of a theme PR.